roles/openshift_common/tasks/main.yml

```diff
@@ -36,7 +36,7 @@
       data_dir: "{{ openshift_data_dir | default(None) }}"
 
 - name: Install the base package for versioning
-  action: "{{ ansible_pkg_mgr }} name={{ openshift.common.service_type }}{{ openshift_version | default('') }} state=present"
+  action: "{{ ansible_pkg_mgr }} name={{ openshift.common.service_type }}{{ openshift_version | default('') }},{{ openshift.common.service_type }}-clients{{ openshift_version | default('') }},{{ openshift.common.service_type }}-master{{ openshift_version | default('') }},{{ openshift.common.service_type }}-node{{ openshift_version | default('') }},{{ openshift.common.service_type }}-sdn-ovs{{ openshift_version | default('') }},tuned-profiles-{{ openshift.common.service_type }}-node{{ openshift_version | default('') }} state=present"
```

Comment:
This role is applied to all masters and nodes, since the issue is with upgrades, should we have a entry in openshift_facts that returns the currently installed packages (possibly minus version number) and use that as a base for the packages to upgrade?